konig-schemagen/src/test/java/io/konig/schemagen/sql/SqlTableGeneratorTest.java

```diff
@@ -145,9 +151,51 @@ public void testGCP() throws Exception {
 		assertTrue(table!=null);
 		}
 		
+	@Test
+	public void testTextField() throws Exception{
+		 AwsShapeConfig.init();
+		 InputStream input = getClass().getClassLoader().getResourceAsStream("sql/SQL-DDL-Text.xlsx");
+		 Workbook book = WorkbookFactory.create(input);
+			Graph graph = new MemoryGraph();
+			NamespaceManager nsManager = new MemoryNamespaceManager();
+			graph.setNamespaceManager(nsManager);
+			
+			WorkbookLoader loader = new WorkbookLoader(nsManager);
+			loader.load(book, graph);
+			input.close();
+			URI shapeId = uri("http://example.com/shapes/MDM_PRODUCT");
+			
+			ShapeManager s = new MemoryShapeManager();
+			
+			ShapeLoader shapeLoader = new ShapeLoader(s);
+			shapeLoader.load(graph);		
+			
+			Shape shape = s.getShapeById(shapeId);
+			assertTrue(shape!=null);
+			ShapeWriter shapeWriter = new ShapeWriter();
+			try {
+				shapeWriter.writeTurtle(nsManager, shape, new File("target/test/sql/MDM_PRODUCT.ttl"));
+			} catch (Exception e) {
+				throw new KonigException(e);
+			}
+			
+			load("target/test/sql/MDM_PRODUCT");
+			
+			assertTrue(shape!=null);
+			
+			SqlTable table = generator.generateTable(shape);
+			
+			assertTrue(table!=null);
+			assertStringField(table, "PPID", 2000);
```

Comment:
We should modify the `assertStringField` method to also verify the `SqlDataype` (CHAR, VARCHAR, TEXT, etc.)